src/lib/analytics/clients/tracks.js

```diff
@@ -85,6 +85,10 @@ export default class Tracks implements AnalyticsClient {
 	}
 
 	send( extraParams: {} ): Promise<Response> {
+		if ( process.env.DO_NOT_TRACK ) {
+			return Promise.resolve( 'tracks disabled per DO_NOT_TRACK variable' );
```

Comment:
Let's add a `debug` call for logging/debugging purposes.

```suggestion
			debug( 'send() => skipping per DO_NOT_TRACK variable' );

			return Promise.resolve( 'tracks disabled per DO_NOT_TRACK variable' );
```